conversion/conversion.go

```diff
@@ -226,8 +239,8 @@ func schemaFromDynamoDB(sampleSize int64) (*internal.Conv, error) {
 
 func dataFromDynamoDB(config spanner.BatchWriterConfig, client *sp.Client, conv *internal.Conv) (*spanner.BatchWriter, error) {
 	mySession := session.Must(session.NewSession())
-	dyclient := dydb.New(mySession)
-
+	cfg := getDynamoDBClientConfig()
+	dyclient := dydb.New(mySession, &cfg)
```

Comment:
I know this is existing code, but having dyclient here and client above seems a bit odd; consider just renaming this to client